Services/JWSProviderInterface.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Lexik\Bundle\JWTAuthenticationBundle\Services;
+
+/**
+ * Interface for JWS provider classes.
```

Comment:
Interface for classes that are able to create and load JSON web signatures (JWS).